tests/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const groupRootId = 'group-root';
```

Comment:
This is unnecessary, and please use `consts.ts` for naming instead. Do you really care about being aligned with other team mates' work?